.github/workflows/test.yml

```diff
@@ -11,12 +11,37 @@ on:
       - main
 
 jobs:
+  define-sessions:
+    runs-on: ubuntu-latest
+    outputs:
+      sessions: ${{ steps.set-sessions.outputs.sessions }}
+    steps:
+      - uses: actions/checkout@v4
+      - uses: actions/setup-python@v5
+        with:
+          python-version: '3.12'
+      - run: pip install nox
+
+      - name: List Nox sessions
```

Comment:
Can you describe the purpose of this step? I don't see `nox_sessions.json` referenced here in the workflow.

Reply:
Oops! The description is, didn't clean up after myself after listing sessions and saving that output in the same step.

78ef01ab1d733c6ae246528e6dff25060f016afc